src/repositories/tweet.repository.ts

```diff
@@ -0,0 +1,37 @@
+import { ObjectId } from "mongodb";
+
+import { TweetModel } from "../models/Tweet";
+
+export const getOneById = (id: ObjectId) => {
+  return TweetModel.findById(id);
+};
+export const createOne = async (
+  authorId: ObjectId,
+  content: string,
+  repliedTo?: ObjectId
```

Comment:
In that place we need to string-type parameters of function. And inside function take ObjectId by string. Only repository has to know how works with database